### PR TITLE
MythicsoftLtd.AgentRansack version 9.0.3289.1

### PR DESCRIPTION
**Pull request by dmzndmzn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/44147)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/44147#event-5987153305

**Review by Trenly — CHANGES_REQUESTED** [review]
This won't be able to be installed unless there is a silent switch available, or until the MSI files can be used.  

Looking at all the installers on their [downloads page](https://www.mythicsoft.com/agentransack/download/) - 
* The MSI files are blocked by #174
* The Portable version is blocked by https://github.com/microsoft/winget-cli/issues/182
* This installer is blocked by https://github.com/microsoft/winget-cli/issues/910

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/MythicsoftLtd/AgentRansack/9.0.3289.1/MythicsoftLtd.AgentRansack.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MythicsoftLtd.AgentRansack
+PackageVersion: 9.0.3289.1
```
Comment (SpecterShell) [inline manifests/m/MythicsoftLtd/AgentRansack/9.0.3289.1/MythicsoftLtd.AgentRansack.installer.yaml:5]:
```suggestion
PackageVersion: 9.0.3289.1
InstallerSwitches:
  Silent: /uin
  SilentWithProgress: /uib
  Log: /logfile"<LOGPATH>"
```

**Comment by denelon** [comment]
@dmzndmzn we've added support for portable packages. This manifest will need to be modified if you would like to add it to the repository.

**Comment by ItzLevvie** [comment]
cc @dmzndmzn

**Review by ItzLevvie — APPROVED** [review]

**Comment by Madhusudhan-MSFT** [comment]
@dmzndmzn - I am investigating on this installer validation failure. It appears the silent installation parameter didn't work as expected for some reason and per our sandbox environment it looks the installation is stuck in the interactive screen

image

Does the silent installation parameter in the manifest works correctly?

**Comment by ItzLevvie** [comment]
Close with reason: Inactive PR;

**Pull request closed without merging**